src/rule_definition_tools.jl

```diff
@@ -203,19 +206,24 @@ function scalar_rrule_expr(f, call, setup_stmts, inputs, partials)
 end
 
 """
-    propagation_expr(Δs, ∂s)
+    propagation_expr(Δs, ∂s, _conj = false)
 
     Returns the expression for the propagation of
     the input gradient `Δs` though the partials `∂s`.
+    Specify `_conj = true` to conjugate the partials.
 """
-function propagation_expr(Δs, ∂s)
+function propagation_expr(Δs, ∂s, _conj = false)
```

Comment:
Do we need still the ability to conjugate?

Reply:
I'm afraid I don't understand the question. Previously `propagation_expr` did the same thing for `frule` and `rrule`. However, we now need to conjugate the partials for the `rrule`, hence introducing this arg, which is set to `true` for the `rrule`. Alternatively, we could create `propagation_frule_expr` and `propagation_rrule_expr` that differ only in whether they conjugate.

Reply:
Yeah, I misunderstood what was going on here. This is totally fine.